test/apps/iodemo/io_demo.cc

```diff
@@ -2385,11 +2385,8 @@ class DemoClient : public P2pDemoCommon {
             server_info_t& server_info   = _server_info[server_index];
             long total_completed         = 0;
             size_t total_bytes_completed = 0;
-            UcxLog *conn_log             = NULL;
-
-            if (opts().per_conn_info) {
-                conn_log = new UcxLog(server_info.conn->get_conn_log_prefix());
-            }
+            UcxLog conn_log(server_info.conn->get_conn_log_prefix(),
+                            opts().per_conn_info);
```

Comment:
Won't destructor dump empty log line with prefix only if `per_conn_info` is disabled?

Reply:
no it prints nothing if `per_conn_info` is disabled.